test/tests/time_steppers/iteration_adaptive/piecewise_linear.i

```diff
@@ -0,0 +1,73 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 50
+  ny = 2
+  xmax = 5
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Functions]
+  [./temp_spike]
+    type = PiecewiseLinear
+    x = '0 1 1.1 1.2 2'
+    y = '1 1 2   1   1'
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+  [./dt]
+    type = TimeDerivative
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = FunctionDirichletBC
+    variable = u
+    boundary = left
+    function = temp_spike
+  [../]
+  [./right]
+    type = NeumannBC
+    variable = u
+    boundary = right
+    value = -1
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = NEWTON
+  start_time = 0.0
+  end_time = 2.0
+  verbose = true
+  [./TimeStepper]
+    type = IterationAdaptiveDT
+    dt = 0.9
+    optimal_iterations = 10
+    timestep_limiting_function = temp_spike
+    max_function_change = 0.5
+  [../]
+[]
+
+[Postprocessors]
+  [./_dt]
```

Comment:
Any good reason for naming it _dt rather than dt?  I've seen that used a number of places, and I'm not sure why.

Reply:
No rationale; just copy/pasted from elsewhere. I fixed it.